wgpu/src/backend/wgpu_core.rs

```diff
@@ -411,23 +411,26 @@ fn map_load_op<V: Default>(op: LoadOp<V>) -> (wgc::command::LoadOp, V) {
     }
 }
 
-fn map_pass_channel<V: Copy + Default>(
+fn map_pass_channel<T: TryFrom<V> + Default, V: Copy + Default>(
     ops: Option<&Operations<V>>,
-) -> wgc::command::PassChannel<V> {
+) -> wgc::command::PassChannel<T>
+where
+    <T as std::convert::TryFrom<V>>::Error: std::fmt::Debug,
```

Comment:
I'm surprised this needs to be fully qualified T::Error doesn't work?

Reply:
It does work, I just usually let compiler generate bounds for me.